ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/ReexecutingExecutionPayloadBlockManager.java

```diff
@@ -92,7 +92,7 @@ protected SafeFuture<?> doStop() {
   public void onSlot(UInt64 slot) {
     super.onSlot(slot);
     pendingBlocksForEPReexecution.removeIf(
-        block -> block.getSlot().isLessThan(slot.plus(EXPIRES_IN_SLOT)));
+        block -> block.getSlot().isLessThan(slot.minusMinZero(EXPIRES_IN_SLOT)));
```

Comment:
we can save the zero check this way:
```suggestion
        block -> block.getSlot().plus(EXPIRES_IN_SLOT).isLessThan(slot);
```

Reply:
Looks much better, thank you